devito/types/tensor.py

```diff
@@ -23,6 +24,41 @@ class TensorFunction(AbstractTensor, Differentiable):
 
     Parameters
     ----------
+    name : str
+        Name of the symbol.
```

Comment:
symbol or TensorFunction?

do we really have "symbol" in the sibling docstrings?